app/scripts/modules/core/src/retry/retry.service.ts

```diff
@@ -22,17 +19,10 @@ export class RetryService {
           if (stopCondition(result)) {
             return result;
           } else {
-            return this.$timeout(interval).then(() =>
-              this.buildRetrySequence(func, stopCondition, limit - 1, interval),
-            );
+            return $timeout(interval).then(() => this.buildRetrySequence(func, stopCondition, limit - 1, interval));
```

Comment:
Should `this` be `RetryService` here?

Reply:
Guessing it's okay because the tests would have caught it if it were wrong, but still curious.

Reply:
🆒